_developer-documentation/extensions.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Extensions
+nav_order: 10
+---
+
+# Extensions
+
+Extensions is an experimental feature. For updates on the progress of extensions, or if you want leave feedback that could help improve the feature, refer to the [issue on GitHub](https://github.com/opensearch-project/OpenSearch/issues/2447).
+{: .warning}
+
+Until _extensions_ were introduced, plugins were the only way to extend OpenSearch functionality. However, plugins have significant shortcomings: they require frequent updates to be up to date with OpenSearch core, they pose a security risk because they run in the same process as OpenSearch, and updating or installing them requires a full cluster restart. Moreover, plugins can fatally impact the cluster in the event of failure.
```

Comment:
```suggestion
Until _extensions_ were introduced, plugins were the only way to extend OpenSearch functionality. However, plugins have significant shortcomings: they require frequent updates to stay up to date with OpenSearch core, they pose a security risk because they run in the same process as OpenSearch, and updating or installing them requires a full cluster restart. Moreover, plugins can fatally impact the cluster in the event of failure.
```
What meaning does _extensions_ have that the concept "Extensions ... an experimental feature" not have? Why introduce italics here but switch back to talking about "Extensions" in line 14? If the italics refer to extensions as an OpenSearch feature, should it be italicized on line 9, too?

Reply:
Hmmm. I see your point. Just wanted to emphasize the name, but can remove.